### PR TITLE
add changelog support

### PR DESCRIPTION
**Pull request by chrsvl**
Do not merge until the client's been merged and deployed to maven. ([PR](https://github.com/appaloosa-store/appaloosa_client/pull/1))

### CODE REVIEW

**Comment by benoittgt** [comment]
LGTM. Maybe telling the user that he can use formated changelog?

File: src/main/resources/org/jenkins/plugins/appaloosa/AppaloosaPublisher/help-changelog.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ The MIT License
```
Comment (rsfez) [inline src/main/resources/org/jenkins/plugins/appaloosa/AppaloosaPublisher/help-changelog.html:2]: Is an MIT License required for an help page?

**Comment by rsfez** [comment]
LGTM

File: src/main/resources/org/jenkins/plugins/appaloosa/AppaloosaPublisher/help-changelog.html (hunk 1 above)
Comment (chrsvl) [reply src/main/resources/org/jenkins/plugins/appaloosa/AppaloosaPublisher/help-changelog.html:2]: I must admit I copy/pasted the description's page...

**Pull request merged**